radio_beam/beam.py

```diff
@@ -429,6 +435,16 @@ def as_kernel(self, pixscale):
                                           self.minor.to(u.deg).value/pixscale,
                                           self.pa.to(u.radian).value)
 
+    def as_struct_element(self, pixscale):
```

Comment:
You mean in general an elliptical top hat? I'm not too sure... that just seemed like the obvious name for it.

I like ```as_tophat_kernel```.